dot/state/block_test.go

```diff
@@ -543,3 +547,53 @@ func TestNumberIsFinalised(t *testing.T) {
 	require.NoError(t, err)
 	require.False(t, fin)
 }
+
+func TestBlockState_GetBestBlockRuntime(t *testing.T) {
+	blockState := newTestBlockState(t, newTriesEmpty())
+	cfg := wasmer.Config{}
+	polkadotRuntimeFilepath, err := runtime.GetRuntime(
+		context.Background(), runtime.POLKADOT_RUNTIME)
+	require.NoError(t, err)
+	code, err := os.ReadFile(polkadotRuntimeFilepath)
+	require.NoError(t, err)
+	runtimeInstance, err := wasmer.NewInstance(code, cfg)
+	require.NoError(t, err)
+	bestBlockHash := blockState.BestBlockHash()
+	blockState.bt.StoreRuntime(bestBlockHash, runtimeInstance)
+
+	resultInstance := blockState.GetBestBlockRuntime()
+	require.Equal(t, runtimeInstance, resultInstance)
+}
+
+func TestBlockState_GetBestBlockRuntime_panic(t *testing.T) {
+	blockState := newTestBlockState(t, newTriesEmpty())
+	defer func() {
+		r := recover()
+		require.Equal(t, "we should always succeed getting the best block runtime but an error occurred: "+
+			"failed to get runtime instance", r)
+	}()
+
+	resultInstance := blockState.GetBestBlockRuntime()
+	require.Equal(t, nil, resultInstance)
+}
+
+func TestBlockState_clearRuntimes(t *testing.T) {
```

Comment:
nit you can run in parallel

```suggestion
func TestBlockState_clearRuntimes(t *testing.T) {
	t.Parallel()

```